packages/mui-material-next/migration.md

```diff
@@ -92,6 +92,46 @@ The following example replaces the `MuiBadge-anchorOriginBottomLeftCircular` cla
 + .MuiBadge-anchorOriginBottomLeft.MuiBadge-overlapCircular
 ```
 
+## Button
+
+See the [ButtonBase](https://github.com/mui/material-ui/blob/master/packages/mui-material-next/migration.md#buttonbase) section for more breaking changes.
+
+## ButtonBase
+
+The breaking changes in this section apply to the following components:
+
+- `Button`
+- `ButtonBase`
+<!-- Add other ButtonBase-based components when those are migrated -->
+
+So the examples below are interchangeable for these components.
+
+### Removed focusRipple
+
+The `focusRipple` prop was removed as ripples are absent in Material You's focused states.
+
+### Prevent default on `key-up` and `key-down` events
+
+If you need to prevent default on a `key-up` and/or `key-down` event, then besides calling `preventDefault` you'll need to set `event.defaultMuiPrevented` to `true` as follows:
+
+```diff
+ const onKeyDown = (event) => {
```

Comment:
For the future, this should be shown on a demo or at least mentioned somewhere in the docs :)

Reply:
Created an issue so we won't forget: https://github.com/mui/material-ui/issues/38581